### DISCUSSION TITLE
How do I get an installer for the most recent version of beemod2?

### DISCUSSION THREAD

**Discussion by ghost** [BEEmod/BEE2-items#4178] [Tech Support / Q&A] [answered]
I feel like I'm Bricking portal 2 by messing with the files. How do I get an installer for the most recent version of beemod2?

**Comment by LautaroL20** [comment, marked as answer]
Make sure to have downloaded and unzipped the latest Packages and Application version of BEE2.4. 
Never use different release versions of the Packages and Application, they always must be from the same Pre-Release Version. Here are the links: [Application](https://github.com/BEEmod/BEE2.4/releases) and [Packages](https://github.com/BEEmod/BEE2-items/releases).
